test/classes/override/type-methods/methodRetType.chpl

```diff
@@ -4,18 +4,18 @@
 //
 
 class Car { var x: int = 0; }
-class Building { var x: int = 0; }
+class Building { var y: int = 0; }
 class Tower: Building { var height: int = 0; }
 
 class Animal {
-  var x: int = 0;
+  var z: int = 0;
```

Comment:
My slight preference is to keep Building and Animal fields called `x`, just to show that it is legal.